### PR TITLE
Adds new volumes syntax

### PR DESCRIPTION
**Pull request by zenorocha**

### CODE REVIEW

**Comment by mdelapenya** [comment]
LGTM 👍 

image

**Comment by zenorocha** [comment]
@sammonsjl I remember you mentioned that we don't need the whole `/opt/liferay/data` directory. Now that we have multiple volumes, which directories do you think we could list?

**Comment by sammonsjl** [comment]
What I did for GA5 was persist the data and osgi directories.  Data will handle document library persistence and osgi will persist osgi state and module deployment allowing for hot deployments after deploying the image.  

Here is the config I used for GA5:

```json
"volumes": {
     "lrdata": "/usr/local/liferay-ce-portal-7.0-ga5/data",
     "osgi": "/usr/local/liferay-ce-portal-7.0-ga5/osgi"
},
```

**Comment by zenorocha** [comment]
Awesome, thanks @sammonsjl!

@mdelapenya can you put a comment on this PR with the changes needed?

File: liferay/wedeploy.json [hunk 1]
```diff
@@ -4,5 +4,7 @@
   "memory": 4096,
   "cpu": 3,
   "dependencies": ["db", "es"],
-  "volume": "/opt/liferay/data"
+  "volumes": {
+    "liferay": "/opt/liferay/data"
```
Comment (mdelapenya) [inline liferay/wedeploy.json]: As mentioned, we should create two volumes:
```json
"volumes": {
   "data": "/opt/liferay/data",
   "osgi": "/opt/liferay/osgi"
}
```

**Pull request merged**